lib/api/bucketPutVersioning.js

```diff
@@ -50,6 +56,16 @@ function _parseXML(request, log, cb) {
     });
 }
 
+function _checkBackendVersioningImplemented(bucket) {
```

Comment:
You can add this function in Config.js

Reply:
This isn't called on the bucket included in the locationConfig, though. It's called on the local S3 bucket